### PR TITLE
Dearrow: allow configuring which elements get dearrowd

### PR DESCRIPTION
**Pull request by Faf4a**
Adds two new options to the `Dearrow` plugin.

- Ability to hide the `Dearrow Toggle` Button
- Ability to only replace certain elements
  - `Titles`, only replaces titles
  - `Thumbnails`, only replaces thumbnails
  - `Everything`, replaces everything (default)
  
---

Preview with each option enabled / disabled:

<details>
<summary>Original</summary>

</details>

<details>
<summary><code>Everything</code></summary>

</details>

<details>
<summary><code>Title</code></summary>

</details>

<details>
<summary><code>Thumbnail</code></summary>

</details>

<details>
<summary><code>Hidden Dearrow Button</code> & <code>Everything</code></summary>

</details>

<details>
<summary>Plugin Settings</summary>

</details>

### CODE REVIEW

File: src/plugins/dearrow/index.tsx [hunk 1]
```diff
@@ -34,11 +35,19 @@ interface Props {
     };
 }
 
+const enum ReplaceElements {
+    ReplaceAllElements,
+    ReplaceTitles,
```
Comment (Vendicated) [inline src/plugins/dearrow/index.tsx:40]:
```suggestion
    ReplaceTitlesOnly,
```

File: src/plugins/dearrow/index.tsx [hunk 2]
```diff
@@ -34,11 +35,19 @@ interface Props {
...
+    ReplaceThumbnails
```
Comment (Vendicated) [inline src/plugins/dearrow/index.tsx:41]:
```suggestion
    ReplaceThumbnailsOnly
```

**Review by Vendicated — COMMENTED** [review]
useful additions! just for more clarity, i suggest these renames

**Comment by Faf4a** [comment]
did the wanted changes 👍🏼

**Comment by Vendicated** [comment]
thank you!

**Pull request merged**